about/index.markdown

```diff
@@ -17,32 +17,38 @@ The primary governance of MoveIt! takes place during monthly meetings of the mai
-Jorge Nicho | SwRI / ROS-I | [jrgnicho](https://github.com/jrgnicho)
-Bence Magyar | Heriot-Watt University | [bmagyar](https://github.com/bmagyar)
-Mike Lautman | PickNik Consulting | [mlautman](https://github.com/mlautman)
-Jon Binney | Iron Ox | [jonbinney](https://github.com/jonbinney)
-Zak Kingston | Rice University | [zkingston](https://github.com/zkingston)
-Henning Kayser | PickNik Consulting | [henningkayser](https://github.com/henningkayser)
+Jorge Nicho        | SwRI / ROS-I | [jrgnicho](https://github.com/jrgnicho)
+Bence Magyar       | Heriot-Watt University | [bmagyar](https://github.com/bmagyar)
+Mike Lautman       | PickNik Consulting | [mlautman](https://github.com/mlautman)
+Jon Binney         | Iron Ox | [jonbinney](https://github.com/jonbinney)
+Zak Kingston       | Rice University | [zkingston](https://github.com/zkingston)
+Henning Kayser     | PickNik Consulting | [henningkayser](https://github.com/henningkayser)
 
 ## Core Contributors
 
-We have a stringent process for giving commit access to the code base to ensure quality, but we want to have as many people involved in reviewing pull requests and responding to issues. We have a non-commit access category of MoveIt! experts we call "Core Contributors" - these participants are recognized for their hard work and contributions to MoveIt! early on. See the [full maintainer policy](maintainer_policy).
+We have a stringent process for giving commit access to the code base to ensure quality, but we want to have as many people involved in reviewing pull requests and responding to issues. We have a non-commit access category of MoveIt! experts we call "Core Contributors" - these participants are recognized for their hard work and contributions to MoveIt! early on and are listed in the [CODEOWNERS](https://github.com/ros-planning/moveit/blob/master/.github/CODEOWNERS) file. See the [full maintainer policy](maintainer_policy).
 
-Name | Organization | GitHub ID
------------- |:------------- |-------------|
-Bryce Willey | Realtime Robotics | [BryceStevenWilley](https://github.com/BryceStevenWilley)
+Name                | Organization  | GitHub ID
+------------------- |:------------- |-------------|
+Bryce Willey        | Realtime Robotics | [BryceStevenWilley](https://github.com/BryceStevenWilley)
 Felix von Drigalski | OMRON SINIC X Corporation | [felixvd](https://github.com/felixvd)
-Simon Schmeißer | isys vision | [simonschmeisser](https://github.com/simonschmeisser)
-William Baker | Houston Mechatronics | [willcbaker](https://github.com/willcbaker)
+Simon Schmeißer     | isys vision | [simonschmeisser](https://github.com/simonschmeisser)
+William Baker       | Houston Mechatronics | [willcbaker](https://github.com/willcbaker)
+Andy Zelenak        | PickNik Consulting | [AndyZe](https://github.com/AndyZe)
+Mohmmad El khzragy  | AUC Egypt | [MohmadAyman](https://github.com/MohmadAyman)
+Kei Okada           | JSK Lab, Univ of Tokyo | [k-okada]((https://github.com/)
```

Comment:
@k-okada please review

Reply:
please change to https//github.com/k-okada

Reply:
There's also a duplicate `(`. So this should be `[k-okada](https://github.com/k-okada/)`. I don't have write access yet, otherwise I'd change it myself.

Reply:
```suggestion
Kei Okada           | JSK Lab, Univ of Tokyo | [k-okada](https://github.com/k-okada)
```